cluster-scope/base/user.openshift.io/groups/argocd-admins/group.yaml

```diff
@@ -7,8 +7,10 @@ users:
   - anishasthana
   - dystewart
   - gregory-pereira
```

Comment:
If we are adding this should we not remove the lower-case versions?

Reply:
not yet because we're still using keycloak for these clusters so when these users log in their usernames will be with lowercase characters. The proper casing doesn't take affect until after we switch to identitatem.